actions/inference-mock/app.py

```diff
@@ -0,0 +1,81 @@
+# Standard
+from dataclasses import dataclass
+import logging
+import pprint
+
+# Third Party
+from completions.completion import create_chat_completion
+from flask import Flask, request  # type: ignore[import-not-found]
+from matching.matching import Matcher
+from werkzeug import exceptions  # type: ignore[import-not-found]
+import click  # type: ignore[import-not-found]
+import yaml
+
+# Globals
+app = Flask(__name__)
+strategies: Matcher  # a read only list of matching strategies
+
+
+# Routes
+@app.route("/v1/completions", methods=["POST"])
+def completions():
+    data = request.get_json()
+    if not data or "prompt" not in data:
+        raise exceptions.BadRequest("prompt is empty or None")
+
+    prompt_debug_str = data["prompt"][:90] + "..."
```

Comment:
This is a tiny issue, not a blocker: this will always add the ellipsis `...` even when `data["prompt"]` is less than 90 characters and the slice did not abbreviate it.